posthog-node/src/feature-flags.ts

```diff
@@ -180,34 +180,36 @@ class FeatureFlagsPoller {
     const payloads: Record<string, JsonType> = {}
     let fallbackToDecide = this.featureFlags.length == 0
 
-    this.featureFlags.map(async (flag) => {
-      try {
-        const matchValue = this.computeFlagLocally(flag, distinctId, groups, personProperties, groupProperties)
-        response[flag.key] = matchValue
-        const matchPayload = await this.computeFeatureFlagPayloadLocally(flag.key, matchValue)
-        if (matchPayload) {
-          payloads[flag.key] = matchPayload
-        }
-      } catch (e) {
-        if (e instanceof InconclusiveMatchError) {
-          // do nothing
-        } else if (e instanceof Error) {
-          this.onError?.(new Error(`Error computing flag locally: ${flag.key}: ${e}`))
+    await Promise.all(
```

Comment:
Slight change in behavior, but I believe the previous was actually broken, we could return `fallbackToDecide` as false because we might not have run the promises before returning.

Reply:
Yeah, I think it was incorrect. Not sure how it ever worked other than by happenstance.